### ISSUE TITLE
Focal length INDI vs MW3

### ISSUE THREAD

**Issue by fenriques** [mworion/MountWizzard3-DIST#14]
hi Michel,
I'm using MW3 RC4, this is more a question than an issue.
When I recently upgraded my system to a longer focal length I updated the value on the MW3 'Config Model Build' tab but then the solver was not able to solve anymore. 
Looking at the FIT keywords of my recorded images, I saw the old focal length value still there. 
That's because I forgot to update the INDI driver value for focal length.
Why is important to have this value on MW3 if the solver uses the INDI one? Users can be misleaded by having two places for this value.
Thanks
Ferrante

**Comment by mworion** [comment]
Hi Ferrante,
You're right. That has been a discussion multiple times. At the beginning (of MW) there was some struggling about having that data in the fits header from imaging software (if you think of using only ASCOM, there is even not a FITs file). On the other hand I got feedbacks after implementing it, that two locations might be problematic, so I came to the current case, if there is data available, I will use it, otherwise I take the entries from MW. But that's not clear in documentation and from the gui. I will improve that.

Michel
